.github/workflows/test-fedora.yaml

```diff
@@ -0,0 +1,96 @@
+name: Libabigail ABI Diff Checks
```

Comment:
Make sure to give this a unique name so it's find-able under https://github.com/woodard/libabigail/actions